src/jdk.compiler/share/classes/com/sun/tools/javac/tree/JCTree.java

```diff
@@ -2013,6 +2013,7 @@ public enum ParameterKind {
         public JCTree body;
         public boolean canCompleteNormally = true;
         public ParameterKind paramKind;
+        public boolean wasMethodReference;
```

Comment:
This is only needed to retain compatibility with some "statistics" option that was added a long time ago. Not sure how much is relevant today, but perhaps better to cleanup in a separate PR if we decide that's no longer needed.